### PR TITLE
{geo}[foss/2023a] Cartopy v0.22.0, Fiona v1.9.5, Cython v3.0.7

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/c/Cython/Cython-3.0.7-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+]
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['fb299acf3a578573c190c858d49e0cf9d75f4bc49c3f24c5a63804997ef09213']
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/Cython/Cython-3.0.7-GCCcore-12.3.0.eb:27]: Move to between toolchain and builddeps lines.

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/Cython/Cython-3.0.7-GCCcore-12.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/c/Cython/Cython-3.0.7-GCCcore-12.3.0.eb]: @akesandgren fixed in dd9ffb19d3

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**